tests/metrics/run.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function check_secondsBehindMaster() {
+	min_val=$1
+	need_cnt=$2
+	$PWD/bin/dmctl.test DEVEL --master-addr=:$MASTER_PORT query-status "test" >$WORK_DIR/query-status.log
+	query_msg=$(cat $WORK_DIR/query-status.log)
+	cnt=$(echo "${query_msg}" | jq -r --arg min_val $min_val '.sources[].subTaskStatus[].sync | select((.secondsBehindMaster|tonumber)>($min_val | tonumber)).secondsBehindMaster' | wc -l)
+	if [ $cnt != $need_cnt ]; then
+		echo "check secondsBehindMaster faild, cnt: $cnt need_cnt: $need_cnt"
+		exit 1
+	fi
+}
+
+function test_syncer_metrics() {
+	# add changeTickerInterval to keep metric from updating to zero too quickly when there is no work in the queue.
+	export GO_FAILPOINTS="github.com/pingcap/dm/syncer/BlockSyncerUpdateLag=return(\"ddl,1\");github.com/pingcap/dm/syncer/changeTickerInterval=return(10)"
```

Comment:
maybe remove the `changeTickerInterval` inject here and increase the batch.

Reply:
when not set changeTickerInterval, lag will update to zero even if current ddl lag is large than zero 😅

so i need lag metrics not updated so quick